packages/client/hmi-client/src/components/widgets/tera-chart-settings-panel.vue

```diff
@@ -69,7 +73,7 @@ const props = defineProps<{
 	generateAnnotation?: (setting: ChartSetting, query: string) => Promise<ChartAnnotation | null>;
 }>();
 
-const emit = defineEmits(['close', 'update-settings-scale', 'delete-annotation', 'create-annotation']);
+const emit = defineEmits(['close', 'update-settings-scale', 'delete-annotation', 'create-annotation', 'change-color']);
```

Comment:
```suggestion
const emit = defineEmits(['close', 'update-settings-scale', 'delete-annotation', 'create-annotation', 'update-settings-color']);
```
Maybe we should rename it so that it's consistent with other existing event name.